### ISSUE TITLE
Keyboard Help in Dula Screen mode

### ISSUE THREAD

**Issue by pathumego** [mattlag/Glyphr-Studio#174]
Keyboard Help icon shows up on Glyph Edit Window on dual screen view, but when clicked it shows up in Menu Window.

**Comment by mattlag** [comment]
Yes, at the moment this is by design.  There are a few (not many) dialogs, and they always show up in the 'main' window (which is the tools area in dual-screen mode).

I'll keep this open, but I think it's lower priority...
